sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -359,40 +357,6 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
-  test("Describe Table with Corrupted Schema") {
```

Comment:
I moved this test to `MetastoreDataSourceSuite`, as now the error will happen when we read the table metadata.